README.md

```diff
@@ -152,8 +152,18 @@ The following properties are accepted:
 
 - `name`
 
+  - _Type_: `string`
+  - _Default value_: undefined
+
   A name to use when displaying the function in the Netlify UI. Populates the `displayName` property in the functions manifest for the specified function.
 
+- `generator`
+
+  - _Type_: `string`
+  - _Default value_: undefined
+
+  A field to use if the function has been autogenerated by a plugin or integration. A recommended format is `@netlify/fake-plugin@1.0.0`, where adding the version is highly appreciated.
```

Comment:
I wonder about this scenario:

A user has a website with an integration at version 1.0.0. The user updates the integration to 1.0.1, and the following build will build the functions, but the functions are identical to the one previously, so we do not upload them. The version in this case will still be 1.0.0, even though 1.0.1 is installed and deploys have happened with this version.

We should check if this is the case. Not a big issue, just something to keep in mind that the version on the function might not be equal the version that is installed.

Reply:
Good point, I'll start looking into this!